src/encoded/tests/test_schema_biosample.py

```diff
@@ -133,3 +133,28 @@ def test_alt_accession_ENCBS_regex(testapp, biosample):
 def test_biosample_organoid_success(testapp, biosample, organoid):
     biosample['biosample_ontology'] = organoid['uuid']
     testapp.post_json('/biosample', biosample, status=201)
+
+
+def test_biosample_post_diffentiation_props(testapp, biosample, organoid):
+    biosample['biosample_ontology'] = organoid['uuid']
+    bio = testapp.post_json('/biosample', biosample).json['@graph'][0]
+    res = testapp.patch_json(
```

Comment:
FYI you can get equivalent behavior without explicit assert statement:

```python
res = testapp.patch_json(
        bio['@id'],
        {'post_differentiation_time': 20},
        expect_errors=True
    )
    assert res.status_code == 422
```

versus

```python
res = testapp.patch_json(
        bio['@id'],
        {'post_differentiation_time': 20},
        status=422
    )
```